apps/team-awesome-web/src/components/team-members/TeamMembers.tsx

```diff
@@ -1,24 +1,32 @@
 import { useContext, useEffect, useState } from "react";
+import { useQuery } from "@apollo/client";
 
-import * as Styled from "./TeamMembers.styled";
+import { Filter } from "../forms/filter";
+import { GET_TEAM_MEMBERS } from "../../graphql/team-members/queries";
+import { Select } from "../forms/select";
+import { StoreContext } from "../store-context";
 import { TeamMember } from "../team-member";
+import * as Styled from "./TeamMembers.styled";
 import * as Types from "../../types";
-import { StoreContext } from "../store-context";
-import { Select } from "../forms/select";
 
 interface TeamMembersProps {
   errorMessage?: string;
   teamMembers: Types.TeamMember[];
 }
 
-export const TeamMembers = ({ ...props }: TeamMembersProps) => {
+export const TeamMembers = ({ errorMessage, ...props }: TeamMembersProps) => {
   const { teamMembers, setTeamMembers } = useContext(StoreContext);
 
-  const [sortedTeamMembers, setSortedTeamMembers] = useState<Types.TeamMember[]>(props.teamMembers);
+  const { loading, data: teamMemberData } = useQuery(GET_TEAM_MEMBERS, {
+    pollInterval: 0,
```

Comment:
You can leave this out. The query does not poll now.

Reply:
Yeah, like I said this morning: I started work on restoring the `useEffect` approach like with the earlier REST requests so this was a leftover to test if everything works (I'm taking the 'regular polling' approach for now)